classes/core/PKPApplication.inc.php

```diff
@@ -224,6 +225,17 @@ public function initializeDatabaseConnection() {
 			error_log("Database query\n$query->sql\n" . json_encode($query->bindings));//\n Bindings: " . print_r($query->bindings, true));
 		});
 
+		Builder::macro('safeCount', function (): int {
```

Comment:
I've decided to add this macro to the Laravel Builder, besides the performance improvement, it's also safer (will work with GROUP BY).